.gitignore

```diff
@@ -18,3 +18,6 @@
 [._]s[a-rt-v][a-z]
 [._]ss[a-gi-z]
 [._]sw[a-p]
+
+# Mac OS auto-generated files
+.DS_store
```

Comment:
note that `.gitignore` is typically case-sensitive, but this should for `.DS_Store` on macos due to its unique FS case sensitivity handling